README.md

```diff
@@ -1,31 +1,33 @@
-# Discord-bot
+# Data Visualisation App. 
 
 
 
 
 
+#### How to run the project:
```

Comment:
This should use a smaller heading.
We need to figure out everything that could go in the readme then organize it accordingly.
\# Data viz site
\#\# Setup
\#\# Use cases
\#\# Other items?

Reply:
yes, we can make that happen later. 

- I added it since, I've not found one quick pick&run project instruction from the last commit, the written instructions would help everyone on the project to run/test at any given commit on `main`. You can update it later as the dev progress.

- I think the README.md development should be collaborative and everybody adds to it. Until the project matures it's hard to settle on a few topics. So we can add stuff incrementally and after a certain point(maybe next week), one can beautify the content in a separate pull request.

Reply:
I agree